### ISSUE TITLE
A way to discover nodes (id's, classes, userData)

### ISSUE THREAD

**Issue by drcmda** [vercel/satori#223]
# Feature Request

In order to identify nodes which later could become controlled. Even forwarding id's would probably be enough

## Description

```jsx
<Satori>
  <div id="test">...

--->

<svg...>
  <g id="test">
    <path ...>
```

**Comment by shuding** [comment]
Good call, I think it should preserve `id`, `class` and `data-*` attributes. There's some work to be done for this because it's not a 1:1 mapping between DOM/SVG elements, e.g. currently Satori will not render an empty and transparent `<div>`.

**Comment by styfle** [comment]
How about passing an allowlist? It might be hard to guess what users will want to preserve so best to let the user configure it.
